pkg/sourcereader/embedded_test.go

```diff
@@ -119,19 +120,20 @@ func (s *MySuite) TestCopyFSToTempDir(c *C) {
 }
 
 func (s *MySuite) TestGetModuleInfo_Embedded(c *C) {
-	ModuleFS = getTestFS()
+	ModuleFS = moduleFS
+	modulereader.ModuleFS = moduleFS
 	reader := EmbeddedSourceReader{}
 
 	// Success
-	moduleInfo, err := reader.GetModuleInfo("modules/network/vpc", tfKindString)
+	moduleInfo, err := reader.GetModuleInfo("modules/test_data/network/vpc", tfKindString)
```

Comment:
Suggestion to make this path less deep, but still clarify that it's a test module:
`"modules/test_role/test_module"`
This removed one level of depth and wouldn't lead to any confusion about the content of the module itself.